.github/workflows/docker-image.yml

```
@@ -0,0 +1,89 @@
+      - name: Check code formatting
+        run: go fmt ./...
+  
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      
+      - name: Install Docker
+        uses: docker/setup-buildx-action@v1
```

actionlint (1.7.7)

the runner of "docker/setup-buildx-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)